bot/exts/utils/snekbox/_cog.py

```diff
@@ -401,6 +401,7 @@ async def send_job(self, ctx: Context, job: EvalJob) -> Message:
                         line_nums=False,
                         mark_no_output=False
                     )
+                    format_text = format_text or "[Empty]"
```

Comment:
And then continuing the previous comment this line also won't be necessary.

Reply:
Refactored to `output_default` str in [dc7653b](https://github.com/python-discord/bot/pull/2326/commits/dc7653b63deee4abcf850b28fc4d30edf04b5e27)